observability/otel/otel-forwarding-logs.adoc

```diff
@@ -0,0 +1,202 @@
+    processors:
+      k8sattributes:
+        auth_type: "serviceAccount"
+        passthrough: false
+        extract:
+          metadata:
+            - k8s.pod.name
+            - k8s.container.name
+            - k8s.namespace.name
+          labels:
+          - tag_name: app.label.component
+            key: app.kubernetes.io/component
+            from: pod
+        pod_association:
+          - sources:
+              - from: resource_attribute
+                name: k8s.pod.name
+              - from: resource_attribute
+                name: k8s.container.name
+              - from: resource_attribute
+                name: k8s.namespace.name
+          - sources:
+              - from: connection
+      resource:
+        attributes: # <1>
+          - key: loki.format # <2>
+            action: insert
+            value: json
+          - key:  kubernetes_namespace_name
+            from_attribute: k8s.namespace.name
+            action: upsert
+          - key:  kubernetes_pod_name
+            from_attribute: k8s.pod.name
+            action: upsert
+          - key: kubernetes_container_name
+            from_attribute: k8s.container.name
+            action: upsert
+          - key: log_type
+            value: application
+            action: upsert
+          - key: loki.resource.labels # <3>
+            value: log_type, kubernetes_namespace_name, kubernetes_pod_name, kubernetes_container_name
+            action: insert
+      transform:
+        log_statements:
+          - context: log
+            statements:
+              - set(attributes["level"], ConvertCase(severity_text, "lower"))
+    exporters:
+      loki:
+        endpoint: https://logging-loki-gateway-http.openshift-logging.svc.cluster.local:8080/api/logs/v1/application/loki/api/v1/push # <4>
+        tls:
+          ca_file: "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+        auth:
+          authenticator: bearertokenauth
+      debug:
+        verbosity: detailed
+    service:
+      extensions: [bearertokenauth] # <5>
+      pipelines:
+        logs:
+          receivers: [otlp]
+          processors: [k8sattributes, transform, resource]
+          exporters: [loki] # <6>
+        logs/test:
+          receivers: [otlp]
+          processors: []
+          exporters: [debug]
+----
+<1> Provides the following resource attributes to be used by the web console: `kubernetes_namespace_name`, `kubernetes_pod_name`, `kubernetes_container_name`, and `log_type`. If you specify them as values for this `loki.resource.labels` attribute, then the Loki Exporter processes them as labels.
+<2> Configures the format of Loki logs. Supported values are `json`, `logfmt` and `raw`.
+<3> Configures which resource attributes are processed as Loki labels.
+<4> Points the Loki Exporter to the gateway of the LokiStack `logging-loki` instance and uses the `application` tenant.
+<5> Enables the BearerTokenAuth Extension that is required by the Loki Exporter.
```

Comment:
```suggestion
<5> Enables the `BearerTokenAuth` extension that is required by the Loki Exporter.
```

Reply:
Thank you for the suggestion. In this case, it's the actual full name of an extension.